main/__tests__/ButtonGroup.test.tsx

```diff
@@ -55,7 +55,7 @@ describe('[ButtonGroup]', () => {
       const {getByText} = renderButtonGroup();
 
       expect(getByText('option 1')).toHaveStyle({
-        color: theme[themeType].text.contrast,
+        color: theme[themeType].text.contrastBasic,
```

Comment:
Please change `contrastBasic` to `contrast`. I think we don't need that many `contrast` color.
Let's remove all contrast colors and leave `contrast` only.

@seongjaeryu FYI.